src/classes/User/User.e2e.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { PassageFlex } from '../PassageFlex';
+import { WebAuthnDevices, WebAuthnType } from './types';
+
+describe('User e2e', () => {
+    const expectedAppId = process.env.TEST_APP_ID ?? 'key';
+    const expectedApiKey = process.env.TEST_API_KEY ?? 'key';
+    const userId = process.env.TEST_USER_ID ?? '';
+    const userExternalId = process.env.TEST_USER_IDENTIFIER ?? '';
+
+    let passage: PassageFlex;
+
+    beforeAll(() => {
+        passage = new PassageFlex({
+            appId: expectedAppId,
+            apiKey: expectedApiKey,
+        });
+    });
+
+    describe('getUser', () => {
+        it('should return the user info', async () => {
+            const actual = await passage.user.get(userExternalId);
+            expect(actual).toStrictEqual({
+                /* eslint-disable @typescript-eslint/no-unsafe-assignment */
+                id: userId,
+                createdAt: expect.any(Date),
+                lastLoginAt: expect.any(Date),
+                loginCount: expect.any(Number),
+                status: expect.any(String),
+                updatedAt: expect.any(Date),
+                userMetadata: expect.any(Object),
+                webauthn: expect.any(Boolean),
+                webauthnDevices: expect.any(Array<WebAuthnDevices>),
+                webauthnTypes: expect.any(Array<WebAuthnType>),
```

Comment:
The other matchers here seem to be using constructors of primitive JS types.
What's the expected behavior of an `expect.any(Array<FooType>)` match?

Reply:
it matches against an array of objects whose shape (keys and key value types) conform to `FooType`